FrontEnd/src/components/authorization/LoginContent.jsx

```diff
@@ -111,16 +101,16 @@ const LoginContent = (props) => {
                   value={enteredEmail}
                   onChange={emailChangeHandler}
                 />
-                <span className={classes["error-message"]}>
-                  {error.errorType === "email" && error.message}
-                </span>
               </div>
+              <span className={classes["error-message"]}>
+                {error.errorType === "email" && error.message}
```

Comment:
consider using [react-hook-form](https://www.npmjs.com/package/react-hook-form) in future to simplify validation work

Reply:
Thanks for recommendation. I'll try to use it next time.